fuel/utils.py

```diff
@@ -87,3 +88,24 @@ def __getstate__(self):
 
         return cls
     return wrap_class
+
+
+def expand_axis_label(axis_label):
+    """Expand an abbreviated axis label.
+
+    Parameters
+    ----------
+    axis_label : str
+        Abbreviated axis label
+
+    """
+    if axis_label == 'b':
+        return 'batch'
+    elif axis_label == 'c':
+        return 'channel'
+    elif axis_label == 't':
+        return 'time'
+    elif re.match('^[0-9]+$', axis_label):
```

Comment:
I'm not sure we need to add numbers to the axes. My idea is that these labels add *semantics*, and "axis 0" is about as non-semantic as it gets.For images the labels could be `w` for width and `h` for height, etc. Otherwise, if there are no semantics, then they should jsut be `f` for features, and this label can be repeated.